src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -252,6 +252,23 @@ public static String formatDateTimeForInstructorHomePage(Date date) {
         return sdf.format(date);
     }
 
+    public static String formatDateTimeForFeedbackSubmissionEditPage(Date date, double timeZone) {
+        StringBuffer formattedDateTime = new StringBuffer(formatTime12H(date));
+        formattedDateTime.append(" (UTC");
+        if (timeZone == 0.0) {
+            formattedDateTime.append(')');
+            return formattedDateTime.toString();
+        }
+        formattedDateTime.append(timeZone < 0 ? " " : " +");
+        int hours = (int) timeZone;
+        double minutesInDecimal = timeZone - hours;
+        formattedDateTime.append(hours)
+                .append(':')
+                .append(minutesInDecimal == 0.5 ? "30" : "00")
```

Comment:
> I think there are timezones where the decimal is 0.25 (15 minutes). 

Should I use a nested ternary for adding in the third comparison? Like so:

``` JAVA
minutesInDecimal == 0.5 ? "30" : minutesInDecimal == 0.25 ? "15" : "00";
```

Or would you recommend multiple if-else statements? Or a map of some sort?

> Also, most of the method (second line onwards) looks very useful, and should be extracted into another method.

Sounds good, will do. 👍